### DISCUSSION TITLE
v1.4.0

### DISCUSSION THREAD

**Discussion by bezo97** [bezo97/IFSRenderer#30] [Announcements]
This version comes with new features and bugfixes, most of them requested by users - thanks for your valuable feedback!   

Highlights:
- Fisheye projection with accompanying [wiki page](https://github.com/bezo97/IFSRenderer/wiki/Create-a-fulldome-planetarium-production) to create planetarium animations.  
- Option to save metadata in exported images.
- A small feature that helps with positioning the focus area precisely.

See the full changelog [here](https://github.com/bezo97/IFSRenderer/blob/master/CHANGELOG.md#140---2024-02-25).

<hr /><em>This discussion was created from the release <a href='https://github.com/bezo97/IFSRenderer/releases/tag/v1.4.0'>v1.4.0</a>.</em>

**Comment by MaxStarovoyt** [comment]
Hello, Bezo97! I've been working on version 1.4.0 for a year, everything was fine until yesterday, when I updated Cinema 4D and Redshift to last version. IFS Renderer stopped launching. Or rather, it starts, hangs for 2 seconds and crashes. not wirk standard and portable version too. I use windows 10. How can I fix it? What could be the problem? Or is there a log file somewhere that reports the cause of the malfunction?

**Reply by bezo97** [reply]
Hi, glad to hear it serves you well. Since you mentioned that the issue started after installing other graphics related programs, I'd first make sure to have the latest graphics drivers and reinstall them. If the issue persists, please [post a bug report](https://github.com/bezo97/IFSRenderer/issues/new?template=bug_report.md) - the template explains how to attach the crashlog.

**Reply by MaxStarovoyt** [reply]
here is my raport: https://github.com/bezo97/IFSRenderer/issues/50

**Reply by MaxStarovoyt** [reply]
Dear bezo97! 
it's just some kind of anomaly. Look. I installed the program on another drive and it works great!!! Solved! 
What was the problem then?
